pkg/models/utils.go

```diff
@@ -32,8 +34,14 @@ func GetDMSData() DMSData {
 	}
 
 	// Available release dates (YYYY-MM)
-	tx.Select("strftime('%Y-%m', release_date) as release_date_text").
-		Group("strftime('%Y-%m', release_date)").Find(&scenes)
+	switch common.DB_TYPE {
+	case "mariadb", "mysql":
+		tx.Select("DATE_FORMAT(release_date, '%Y-%m') as release_date_text").
+			Group("DATE_FORMAT(release_date, '%Y-%m')").Find(&scenes)
+	case "sqlite3":
+		tx.Select("strftime('%Y-%m', release_date) as release_date_text").
+			Group("strftime('%Y-%m', release_date)").Find(&scenes)
+	}
```

Comment:
I'd drop that filter and replace it with proper between-dates filter (requires UI changes ofc)

Reply:
This would break DLNA release date filtering so I left it.